test/unit/sql/data-types.test.js

```diff
@@ -1062,8 +1062,10 @@ describe(Support.getTestDialectTeaser('SQL'), () => {
 
       testsql('FLOAT(11)', DataTypes.FLOAT(11), {
         default: 'FLOAT(11)',
-        postgres: 'FLOAT(11)', // 1-24 = 4 bytes; 35-53 = 8 bytes
-        mssql: 'FLOAT(11)' // 1-24 = 4 bytes; 35-53 = 8 bytes
+        postgres: 'FLOAT(11)',
+        // 1-24 = 4 bytes; 35-53 = 8 bytes
```

Comment:
This comment should go above or beside `postgres` statement

Reply:
heh, `recast` isn't very good at preserving comment locations...I had corrected most of the places this happened but missed this one